server/src/test/java/pl/kejbi/tin/TinApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package pl.kejbi.tin;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class TinApplicationTests {
```

Comment:
tu będa jakieś testy integracyjne całej aplikacji? Jeżeli nie to można usunać ten folder

Reply:
domyślnie dodany plik podczas tworzenia projektu, można usunąć